README.md

```diff
@@ -45,7 +46,7 @@ On some systems you might have to use pip3 instead of pip in order to use Python
 
 
 ## Dependencies
-  * python 3.6+
+  * python 3.6+ (3.10 for latest)

```

Comment:
This will soon be 3.10 or 3.12, so might as well change it to that and not mention 3.6.

Reply:
Did it